pallets/accounts/src/lib.rs

```diff
@@ -496,6 +517,52 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Update 'data'
+		///
+		/// TODO
+		#[pallet::weight(25_000)] // TODO #1 - weight
+		pub fn sponsor_account(origin: OriginFor<T>, external_id: ExternalID) -> DispatchResult {
```

Comment:
Looks good. We will document this later.

Reply:
Yeah sorry, this is what the discord bot uses 👯